client/state/comments/reducer.js

```diff
@@ -108,6 +113,7 @@ export function items( state = {}, action ) {
 
 	return state;
 }
+items.schema = commentsSchema;
```

Comment:
@blowery was there any reason why we didn't persist comments before? Were we afraid of running out of space?